docs/1_introduction.md

```diff
@@ -0,0 +1,43 @@
+# 💉 Introduction to the [ReVanced patcher](https://github.com/revanced/revanced-patcher)
+
+Familiarize yourself with the [ReVanced patcher](https://github.com/revanced/revanced-patcher).
```

Comment:
Context is already known.
```suggestion
Familiarize yourself with the [Patcher framework](https://github.com/revanced/revanced-patcher).
```

Reply:
For consistency, whenever a repository is referenced, the repository name is used. "Patcher framework" is too arbitrary.

Reply:
That's understandable.